actors/builtin/miner/miner_actor.go

```diff
@@ -1025,6 +1027,8 @@ func (a Actor) TerminateSectors(rt Runtime, params *TerminateSectorsParams) *Ter
 			removedPower, err := deadline.TerminateSectors(store, currEpoch, byPartition, info.SectorSize, quant)
 			builtin.RequireNoErr(rt, err, exitcode.ErrIllegalState, "failed to terminate sectors in deadline %d", dlIdx)
 
+			st.EarlyTerminations.Set(dlIdx)
```

Comment:
FYI @Stebalien here's another of those bugs where miner actor code broke consistency of different bits of the state. #828.